### PR TITLE
[IA-3862] add css rules to change line breaking in table and environments

### PR DESCRIPTION
**Pull request by ghost**
Fixed overflow that was occuring when windows were too narrow in the cloud environments list table. Now all tables are scalably themselves, using the EM property to ensure that sizing is consistent across all platforms. Additionally, all overflowing tables now scroll to the side.

https://user-images.githubusercontent.com/123021647/214692586-9c78b6bc-7403-4fc7-b5ab-1978cdcf2659.mov

### CODE REVIEW

**Comment by lucymcnatt** [comment]
image
just noticed the status section under persistent disks looks odd, the 's' is cut off and ready is wrapped

**Comment by ghost** [comment]
alright fixed that, just needed to increment a width by 2em

**Comment by n8landolt** [comment]
Looks great! 

Could you adjust the size of disk status so the word "Deleting" doesn't wrap?

Screenshot 2023-01-25 at 4 35 03 PM

**Comment by n8landolt** [comment]
Also could we try to have these date columns be similar in size? The top one looks better formatted than the bottom.

Screenshot 2023-01-25 at 4 37 34 PM

File: src/pages/Environments.js [hunk 1]
```diff
@@ -594,13 +594,13 @@ export const Environments = ({ nav = undefined }) => {
           span({ style: { fontWeight: 600 } }, [' Hide resources you did not create'])
         ])
       ]),
-      runtimes && h(SimpleFlexTable, {
+      runtimes && div({ style: { overflow: 'scroll' } }, [h(SimpleFlexTable, {
```
Comment (n8landolt) [inline src/pages/Environments.js]: Nit: bring h(SimpleFlex...) to the next line, and add a new line to separate the `])` for the div below (Current line 827)

**Review by n8landolt — APPROVED** [review]
Just the two Nit (nitpicky) changes.

**Review by lucymcnatt — APPROVED** [review]

**Pull request merged**